src/init.cpp

```diff
@@ -440,7 +440,7 @@ void SetupServerArgs(ArgsManager& argsman)
                  " If <type> is not supplied or if <type> = 1, indexes for all known types are enabled.",
                  ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
 
-    argsman.AddArg("-addnode=<ip>", strprintf("Add a node to connect to and attempt to keep the connection open (see the addnode RPC help for more info). This option can be specified multiple times to add multiple nodes; connections are limited to %u at a time and are counted separately from the -maxconnections limit.", MAX_ADDNODE_CONNECTIONS), ArgsManager::ALLOW_ANY | ArgsManager::NETWORK_ONLY, OptionsCategory::CONNECTION);
+    argsman.AddArg("-addnode=<ip>[=manual-block-relay]", strprintf("Add a node to connect to and attempt to keep the connection open (see the addnode RPC help for more info). This option can be specified multiple times to add multiple nodes; connections are limited to %u at a time and are counted separately from the -maxconnections limit. Append =manual-block-relay to disable transaction and address relay.", MAX_ADDNODE_CONNECTIONS), ArgsManager::ALLOW_ANY | ArgsManager::NETWORK_ONLY, OptionsCategory::CONNECTION);
```

Comment:
Prefer `=blocksonly` to be clearer and match the existing `-blocksonly` option.

Reply:
`-blocksonly` mode is different from block relay peers.

Reply:
How so? Maybe it needs better docs on the difference...

Reply:
Here you go: https://bitcoin.stackexchange.com/questions/112828/what-is-a-block-relay-only-connection-what-is-it-used-for

Reply:
That doesn't contrast.

Reply:
Some differences:
- -blocksonly is a global startup option (applying to all connections) -block-relay-only is connection-specific
- A node in -blocksonly mode participates in address relay, we never send out addresses on -block-relay-only connections (and ignore incoming ones).
- A node in -blocksonly signals not to receive any transactions (`fRelay`) and will disconnect if that is breached, but it may in special cases send out transactions itself (if submitted via RPC). We'll  never send a tx over a block-relay-only connection.
- A node in -blocksonly mode still has two of their outbound connections as -block-relay-only (for which then the block-relay-only rules apply).